src/sub-record/sub-record.component.ts

```diff
@@ -57,8 +57,12 @@ export class SubRecordComponent extends AbstractTrackerComponent implements OnCh
   constructor(public jsonStoreService: JsonStoreService,
     public appGlobalsService: AppGlobalsService,
     public tabsUtilService: TabsUtilService,
+    public changeDetectorRef: ChangeDetectorRef,
     public keysStoreService: KeysStoreService) {
     super();
+    this.appGlobalsService.adminModeSubject.subscribe(adminMode => {
```

Comment:
Is this needed? Since you already `markForCheck` in primitive-field-component (which is a child of this component) this should be checked as well.